.pre-commit-config.yaml

```diff
@@ -6,6 +6,6 @@ repos:
     -   id: check-merge-conflict
     -   id: check-symlinks
 -   repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 23.1.0
```

Comment:
According to the black stability policy, any version 23.*.* should be fine, and this allows bug fixes: https://black.readthedocs.io/en/stable/the_black_code_style/index.html#stability-policy

Reply:
Should this be changed to "rev: 23"?

Reply:
It doesn't work without the full version number.